plugins/wp/index.js

```diff
@@ -87,7 +109,9 @@ module.exports = function (app) {
         modified: moment(post.modified).format('Do MMMM YYYY'),
         image: post.featured_image,
         thisPageFullUrl: req.protocol + '://' + req.get('host') + req.originalUrl,
-        categories: post.categories ? Object.keys(post.categories) : []
+        categories: post.categories ? Object.keys(post.categories) : [],
+        caption: attachments_caption.join(''),
```

Comment:
@mpolidori 

I'd recommend not to join the strings (although right now it's always a single string in the array). 

Maybe it's better to pass all the attachments as an array, then in the nunjucks template use them as needed (if need only the first attachment - then ok, take only first)

```javascript
attachments: [
  {
    "ID": 43602,
    "URL": "",
    "alt": "",
    "caption": "",
  },
]
```

Alternatively (and I'd vote for this one), if we only need the featured image alt and caption pass like those (didn't test the code below, just as an idea):

```javascript
const featuredImageAttachment = Object
  .values(post.attachments)
  .find(attachment => attachment.URL === post.featured_image)

{
  ...
  image: post.featured_image, // already there
  imageCaption: featuredImageAttachment.caption,
  imageAlt: featuredImageAttachment.alt,
  ...
}
```

Reply:
also please add checks for nulls if using the above suggestion, because `featuredImageAttachment.alt` will throw an error if featuredImageAttachment is null or undefined

Reply:
@EvgeniiaVak So, this appears to prevent any breakage when I pass false fields or values:

```
      const postAttachments = post.attachments ? Object.values(post.attachments) : []
      const featuredImageAttachments = Object
        .values(postAttachments)
        .find(attachment => attachment.URL === post.featured_image) || []
```

What do you think? I already have fallback values in the templates in case caption or alt are null

Reply:
I read in a few places that using `||` like that might be bad practice, but I'm not sure that it's a bad idea here

Reply:
@mpolidori you shouldn't add `|| []` after `.find(attachment => attachment.URL === post.featured_image)` (find returns an object, not an array, see example here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)

Reply:
so it should be something like: 

```javascript
      const featuredImageAttachment = Object
        .values(postAttachments)
        .find(attachment => attachment.URL === post.featured_image) || {}
```

Reply:
The idea to use `|| <something>` is fine I think :+1:

Reply:
@EvgeniiaVak Ah, good call. Okay, yeah, just pushed the new change (obviously still working locally with false/null cases)